src/components/Libraries.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext'
+
+export default function Libraries() {
+  const { siteConfig } = useDocusaurusContext()
+
+  const tableRow = (data) => {
+    return data.map(({ name, desc, link, version }) => {
+      return (
+        <tr key={name}>
+          <td>
+            <strong>{name}</strong>
+            <br />
+            {desc}
+          </td>
+          <td class='cell-center'>{version.includes(2) ? '✅' : '❌'}</td>
+          <td class='cell-center'>{version.includes(1) ? '✅' : '❌'}</td>
```

Comment:
This breaks down if we ever move to 2.1 or something like that. Can we make it a bit more robust? Maybe check for the dot as well, and see if the version starts with, not only includes the number?

Reply:
What's our versioning strategy?
Because this table might start looking ridiculous after 5 versions.

Reply:
It all depends on how massive the change recommendations are after the browser implementation we're experimenting with. The first time we did this, we've had to move from v1 to v2. If the changer aren't big, we'll probably do v2.1, and that's going to be the only "2" version we'll talk about. So the table might be "v2.1, v1" without mentioning v2.0